### DISCUSSION TITLE
Github student devloper pack

### DISCUSSION THREAD

**Discussion by VarshaUN** [community/community#144381] [GitHub Education]
I tried giving my ID pic as a proof but it's not acccepting, to try with even email id is not issued by my university.  What to do? 

Thanks

**Comment by rajputdivyanshu81** [comment]
They accept only mail ID not pic

**Comment by jonhubby** [comment]
Hey Varsha, I ran into something similar before. GitHub usually requires a valid school-issued email address to verify student status. If your university doesn’t provide one, you can try submitting other official documents like a student ID, along with a current enrollment letter or fee receipt. Just uploading a photo alone might not be enough.

You can also reach out to GitHub Education Support directly; they’re usually helpful in reviewing alternative documents. Hope that helps!

**Comment by salmanalfarisi11** [comment]
You don’t need a school-issued email—just any one of these official proofs of current enrollment (make sure it clearly shows your name, institution, and date):

1.Photo of your student ID with an enrollment/expiry date
2. Class schedule or course registration page
3. Academic transcript
4. Enrollment or affiliation verification letter 

Tips:

1. Use a straight-on scan or high-res photo (all corners visible).
2. Save as JPG/PNG within the uploader’s size limits.
3. If digital-only (PDF/transcript), take a clean screenshot or photo.

If it still rejects, file a ticket via GitHub Education Support and attach whichever document you have—someone will manually verify.

**Comment by Rknilkant** [comment]
Hi there,

I understand someone is facing issues getting their GitHub Student Developer Pack approved, even after providing a photo ID, and that their university email is not issued or recognized.

When I went through the process myself, here’s what worked for me and may help in this case:

A clear photo of your student ID — ideally showing your name and either an enrollment date or expiration date.

Alternatively, or in addition, a class schedule, course registration page, or official enrollment confirmation from your institution.

If your university does not provide a .edu or recognized domain email, you can still apply with a personal email but must ensure the supporting documents clearly verify your student status.

Make sure that:

All documents are recent and clearly legible.

You upload documents in English or provide a translated version if needed.
